lib/modules/apostrophe-pages/lib/api.js

```diff
@@ -1551,7 +1551,9 @@ module.exports = function(self, options) {
         return setImmediate(callback);
       },
       findExisting: function(callback) {
-        return self.find(req, { slug: item.slug }).published(null).permission(false).trash(null).toObject(function(err, _existing) {
+        // Tolerate presence or absence of a trailing slash on the
+        // existing page, to reduce false negatives
+        return self.find(req, { slug: new RegExp('^' + self.apos.utils.regExpQuote(item.slug.replace(/\/$/, '')) + '/?$') }).published(null).permission(false).trash(null).toObject(function(err, _existing) {
```

Comment:
What if you make a new var, something like...
```javascript
slashless_slug = new RegExp('^' + self.apos.utils.regExpQuote(item.slug.replace(/\/$/, '')) + '/?$') });
return self.find(req, { slug: slashless_slug }).published(null).permission(false).trash(null).toObject(function(err, _existing) {
```
and that would eliminate the need for the comment.